pyscitt/pyscitt/client.py

```diff
@@ -15,22 +16,31 @@
 from cryptography.hazmat.primitives.serialization import load_pem_private_key
 from loguru import logger as LOG
 
-from . import crypto
 from .governance import GovernanceClient
 from .prefix_tree import PrefixTreeClient
 from .receipt import Receipt
 
 CCF_TX_ID_HEADER = "x-ms-ccf-transaction-id"
 
+
+class MemberAuthenticationMethod(ABC):
+    @property
+    @abstractmethod
+    def get_key_id(self) -> bytes:
+        pass
+
+    @abstractmethod
+    def sign(self, data: bytes) -> bytes:
+        pass
+
+
 # copied from CCF/tests/infra/clients.py
 class HttpSig(httpx.Auth):
     requires_request_body = True
```

Comment:
```suggestion
class HttpSig(httpx.Auth):
    member_auth_client: MemberAuthenticationMethod

    requires_request_body = True
```

It's a good idea to have the field type.